docs/design/features/globalization-hybrid-mode.md

```diff
@@ -287,6 +286,15 @@ Affected public APIs:
 
 Web API does not expose locale-sensitive indexing function. There is a discussion on adding it: https://github.com/tc39/ecma402/issues/506. In the current state, as a workaround, locale-sensitive string segmenter combined with locale-sensitive comparison is used. This approach, beyond having the same compare option limitations as described under **String comparison**, has additional limitations connected with the workaround used. Information about additional limitations:
 
+- Methods that calculate `matchLength` always return throw PlatformNotSupported exception:
+
+[CompareInfo.IndexOf](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.compareinfo.indexof?view=system-globalization-compareinfo-indexof(system-readonlyspan((system-char))-system-readonlyspan((system-char))-system-globalization-compareoptions-system-int32@))
+
+[CompareInfo.LastIndexOf](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.compareinfo.lastindexof?view=system-globalization-compareinfo-lastindexof(system-readonlyspan((system-char))-system-readonlyspan((system-char))-system-globalization-compareoptions-system-int32@))
+
+- String.Replace that uses `StringComparison` argument relies internally on IndexOf with `matchLength` argument. From this reason, it throws PlatformNotSupportedException:
+[String.Replace](https://learn.microsoft.com/en-us/dotnet/api/system.string.replace?view=system-string-replace(system-string-system-string-system-stringcomparison))
```

Comment:
We need to add this also under **String starts with / ends with** section yes?

Reply:
We have it
browser:
https://github.com/dotnet/runtime/blob/7f315b171c47df5255f03def1a389369688bc92c/docs/design/features/globalization-hybrid-mode.md?plain=1#L271

ios:
https://github.com/dotnet/runtime/blob/7f315b171c47df5255f03def1a389369688bc92c/docs/design/features/globalization-hybrid-mode.md?plain=1#L449

Reply:
Thanks, missed that :)